src/test/java/racingcar/CarsTest.java

```diff
@@ -31,48 +35,43 @@ void drive() {
     @Test
     @DisplayName("canDrive 반환값이 false라면 전진하지 않는다")
     void notDrive() {
-        Cars cars = new Cars(CAR_NAMES);
-        cars.drive(() -> false);
+        cars.drive(NEVER_MOVE);
         List<Integer> drivingDistances = cars.drivingDistances();
 
         assertThat(drivingDistances).containsExactly(0, 0, 0);
     }
 
     @Test
     @DisplayName("-의 개수를 이용해 주행 거리를 표현한다")
-    void getResult() {
-        Cars cars = new Cars(CAR_NAMES);
-
-        cars.drive(() -> true);
-        assertThat(cars.getResult()).isEqualTo(
-            String.format("%s : -\n%s : -\n%s : -", CAR_NAMES[0], CAR_NAMES[1], CAR_NAMES[2]));
-
-        cars.drive(() -> true);
-        assertThat(cars.getResult()).isEqualTo(
-            String.format("%s : --\n%s : --\n%s : --", CAR_NAMES[0], CAR_NAMES[1], CAR_NAMES[2]));
+    void result() {
+        cars.drive(ALWAYS_MOVE);
+        assertThat(cars.result()).isEqualTo(
+            String.format("%s : -\n%s : -\n%s : -", CAR_NAMES.get(0), CAR_NAMES.get(1), CAR_NAMES.get(2)));
```

Comment:
CarFixture가 있다면 index를 활용하지 않고 검증할 수 있을 것 같아요
```suggestion
        cars.drive(ALWAYS_MOVE);
        assertThat(cars.result()).isEqualTo(
            String.format("%s : -\n%s : -\n%s : -", CarFixture.K3, CarFixture.K5, CarFixture.K9));
```